From 16c96edd16ddb2a92c7ac1423463636473e22859 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Wed, 20 Apr 2011 11:36:05 +0800
Subject: [PATCH 3/7] ENGR00133601-1 Change mkfs.ubifs makefile for
 gingerbread 64bit build

on 64bit os machine, we do not have ia32 .a libraries, so
build mkfs.ubifs with share libraries.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 mkfs.ubifs/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mkfs.ubifs/Android.mk b/mkfs.ubifs/Android.mk
index 531a916..512b562 100644
--- a/mkfs.ubifs/Android.mk
+++ b/mkfs.ubifs/Android.mk
@@ -17,7 +17,7 @@ LOCAL_SRC_FILES := \
 LOCAL_CFLAGS = -O2 -Wall
 LOCAL_CFLAGS+= -Wpointer-arith -Wwrite-strings -Wstrict-prototypes
 LOCAL_CFLAGS+= -Wmissing-prototypes -Winline
-LOCAL_LDLIBS:= -lz -llzo2 -lm -luuid -static
+LOCAL_LDLIBS:= -lz -llzo2 -lm -luuid
 
 LOCAL_C_INCLUDES += $(LOCAL_PATH)/
 LOCAL_C_INCLUDES += $(LOCAL_PATH)/hashtable
-- 
1.8.0

